pyso/stack_search_module.py

```diff
@@ -8,7 +8,7 @@
 import async_timeout
```

Comment:
You should sort your imports according to PEP8:

> Imports should be grouped in the following order:
> - standard library imports
> - related third party imports
> - local application/library specific imports
> You should put a blank line between each group of imports.

[More about import style here](https://www.python.org/dev/peps/pep-0008/#imports)